src/app/pages/public/LivesPage/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button, Table } from 'antd';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router';
+import { useListLivesQuery } from '../../../api';
+import InternalErrorResult from '../../../components/results/InternalErrorResult';
+import BasicLayout from '../../../layouts/BasicLayout';
+
+const LivesPage = () => {
+    const { t } = useTranslation(['life']);
+    const navigate = useNavigate();
+
+    const { data, loading, error } = useListLivesQuery();
+
+    if (error) {
+        return <InternalErrorResult />;
+    }
+
+    const response = data?.listLives;
+
+    const columns = useMemo(
+        () => [
+            {
+                title: t('firstName'),
+                dataIndex: 'fullName',
+                key: 'fullName',
+            },
+            {
+                title: t('birthday'),
+                dataIndex: 'birthday',
+                key: 'birthday',
+            },
+            {
+                title: t('title'),
+                dataIndex: 'title',
+                key: 'title',
+            },
+            {
+                title: t('hobbies'),
+                dataIndex: 'hobbies',
+                key: 'hobbies',
+                render: hobbies => hobbies.map(hobby => <p>{hobby.toUpperCase()}</p>),
```

Comment:
same comment here for the `key` property

Reply:
Fixed. Will include in next commit (part 5)